app/pages/admin/project-status.jsx

```diff
@@ -188,7 +188,7 @@ class ProjectStatus extends Component {
         <div className="project-status__section">
           <h4>Information</h4>
           <ul>
-            <li>Id: {this.state.project.id}</li>
+            <li>Id: <a href={"/lab/" + this.state.project.id}>{this.state.project.id}</a></li>
```

Comment:
Not a blocking issue, but we've switched to using backticks for string interpolation
```javascript
{`/lab/${this.state.project.id}`}
```